### PR TITLE
Hit rate in queue charts instead of total review transactions

### PR DESCRIPTION
**Pull request by irinatarn**
queues

### CODE REVIEW

**Review by haixiehx — APPROVED** [review]
Looks good, approving.

**Comment by satyamamit** [comment]
Chart title says Total review while we are showing hit rate in the chart.
image

File: frontend/src/screens/review-console/item-details/parts/transaction-history/transaction-history.tsx [hunk 1]
```diff
@@ -276,7 +278,7 @@ export class TransactionHistory extends Component<TransactionHistoryProps, never
                 title="Transaction history"
             >
                 <MessageBar className={`${CN}__message-bar`}>
-                    The table contains full information only for last 10 transactions / last week transactions.
+                    The table contains full information only for last 50 transactions / last week transactions.
```
Comment (satyamamit) [inline frontend/src/screens/review-console/item-details/parts/transaction-history/transaction-history.tsx:281]: did we change the numbers here, as previously we were getting everything but was showing just last 10. I didn't see code where where you changed from 10 to 50.

**Review by satyamamit — APPROVED** [review]
Please verify there max transaction limit in code once.

File: frontend/src/screens/review-console/item-details/parts/transaction-history/transaction-history.tsx (hunk 1 above)
Comment (irinatarn) [reply frontend/src/screens/review-console/item-details/parts/transaction-history/transaction-history.tsx:281]: We don't have that restriction. We don't limit to 10.

**Pull request merged**